juniper_codegen/src/impl_object.rs

```diff
@@ -112,6 +114,16 @@ pub fn build_object(args: TokenStream, body: TokenStream, is_internal: bool) ->
                     ),
                 };
 
+                if !attrs.arguments.is_empty() {
+                    let deprecation_warning = vec![
+                        "Setting arguments via #[graphql(arguments(...))] on the method",
+                        "is deprecrated. Instead use #[graphql(...)] as attributes directly",
+                        "on the arguments themselves.",
+                    ]
+                    .join(" ");
+                    eprintln!("{}", deprecation_warning);
```

Comment:
It would be nice if [proc-macro-error](https://crates.io/crates/proc-macro-error) supported warnings.